commit e87ca9d21bf962cbf737bf55679af3371c81317b (refs/changes/42/3242/1)
Author: Dan McDonald <danmcd@joyent.com>
Date:   2018-01-22T10:00:58-05:00 (1 year, 9 months ago)
    
    NAPI-440 IP address object needs belongs_to_type of router

diff --git a/lib/models/nic/common.js b/lib/models/nic/common.js
index 71f4c7e..3b17731 100644
--- a/lib/models/nic/common.js
+++ b/lib/models/nic/common.js
@@ -30,7 +30,7 @@ var validate = require('../../util/validate');
 // --- Globals
 
 var BUCKET = require('./bucket').BUCKET;
-var BELONGS_TO_TYPES = [ 'other', 'server', 'zone' ];
+var BELONGS_TO_TYPES = [ 'other', 'server', 'zone', 'router' ];
 var VALID_NIC_STATES = [ 'provisioning', 'stopped', 'running' ];
 
 var getPoolIntersections = util_intersect.getPoolIntersections;
